operator/internal/controller/redpanda/redpanda_controller_test.go

```diff
@@ -291,21 +365,34 @@ func (s *RedpandaControllerSuite) minimalRP(useFlux bool) *redpandav1alpha2.Redp
 
 	return &redpandav1alpha2.Redpanda{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: name,
+			Name: name, // GenerateName doesn't play nice with SSA.
 		},
 		Spec: redpandav1alpha2.RedpandaSpec{
 			ChartRef: redpandav1alpha2.ChartRef{
 				UseFlux: ptr.To(useFlux),
 			},
 			ClusterSpec: &redpandav1alpha2.RedpandaClusterSpec{
-				Image: &redpandav1alpha2.RedpandaImage{
-					Repository: ptr.To("redpandadata/redpanda"), // Override the default to make use of the docker-io image cache.
-				},
+				Config: &redpandav1alpha2.Config{},
```

Comment:
Why empty Config struct is added?

Reply:
Oh, sorry that leaked in from another PR. It's so that I can set config later on without having to worry about nil pointers. Added a comment.